src/go/k8s/pkg/resources/configuration/patch.go

```diff
@@ -76,6 +79,24 @@ func ThreeWayMerge(
 	}
 	for k, v := range apply {
 		if oldValue, ok := current[k]; !ok || !PropertiesEqual(log, v, oldValue, schema[k]) {
+			// Value must be type []interface{} if array
+			// https://github.com/redpanda-data/team-kubernetes-internal/issues/11
```

Comment:
This is public repo. I would like to not include internal issue in this repo. Could you just provide more context in this comment from that issue.